### DISCUSSION TITLE
UseRenderHook and ControlCode not available

### DISCUSSION THREAD

**Discussion by AleVin76** [spectreconsole/spectre.console#1373] [Q&A] [answered]
**Information**
 - OS: Win11 Pro

I created a console project where I added the code from the Progress example, but the UseRenderHook and ControlCode elements are not available.

I attach the solution.
[ConsoleApp2.zip](https://github.com/spectreconsole/spectre.console/files/13439383/ConsoleApp2.zip)

**Comment by patriksvensson** [comment]
I'm moving this over to our Q/A discussion forum. We do not open attached zip files, so you will need to create a GitHub repository and link to that.

**Reply by AleVin76** [reply]
https://github.com/AleVin76/ConsoleApp2

**Comment by patriksvensson** [comment, marked as answer]
You will need to upgrade Spectre.Console to 0.48 or use a prerelease version of 0.47 for your program to compile.

**Reply by AleVin76** [reply]
Thanks for your help..
